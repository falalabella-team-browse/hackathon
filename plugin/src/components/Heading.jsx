import React from "react";
import styled from "styled-components";

const Heading1 = styled.h1`
  border-bottom: 2px solid #aad500;
  font-weight: 400;
  font-size: 20px;
  padding-bottom: 8px;
`;

const H1 = ({ children }) => {
  return <Heading1>{children}</Heading1>;
};


const Heading = {
  H1,
};

export default Heading;
