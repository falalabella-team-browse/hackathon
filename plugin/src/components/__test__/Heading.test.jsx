import React from "react";
import { shallow } from "enzyme";
import Heading from "../Heading";

describe("Heading", () => {
  it("H1 should render properly", () => {
    const wrapper = shallow(<Heading.H1>this is h1</Heading.H1>);

    expect(wrapper.debug()).toMatchSnapshot();
  });

  it("H2 should render properly", () => {
    const wrapper = shallow(<Heading.H2>this is h2</Heading.H2>);

    expect(wrapper.debug()).toMatchSnapshot();
  });

  it("H4 should render properly", () => {
    const wrapper = shallow(<Heading.H4>this is h1</Heading.H4>);

    expect(wrapper.debug()).toMatchSnapshot();
  });
});
