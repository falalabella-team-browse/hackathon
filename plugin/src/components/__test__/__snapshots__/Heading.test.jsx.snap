// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading H1 should render properly 1`] = `
"<Heading__Heading1>
  this is h1
</Heading__Heading1>"
`;

exports[`Heading H2 should render properly 1`] = `
"<Heading__Heading2>
  this is h2
</Heading__Heading2>"
`;

exports[`Heading H4 should render properly 1`] = `
"<Heading__Heading4>
  this is h1
</Heading__Heading4>"
`;
